moveit_planners/chomp/chomp_motion_planner/src/chomp_trajectory.cpp

```diff
@@ -208,12 +208,12 @@ void ChompTrajectory::fillInMinJerk()
 {
   double start_index = start_index_ - 1;
   double end_index = end_index_ + 1;
-  double T[6];  // powers of the time duration
```

Comment:
I suggest to just rename `T` to `t`. Powers is a kind of misleading name here, but `t[i] = t^i`.

Reply:
Agree, I think `t` or `time` is a better name here.

Reply:
I agree `powers` is not a good name. But `t` is used in the following code block as `//powers of the time index point`, so I think maybe it's better to name `T` here as `td` and the following `t` as `ti` to distinguish them.

Reply:
Agreed.